cmake/modules/FindFmt.cmake

```diff
@@ -12,6 +12,19 @@
 #
 #   Fmt::Fmt   - The Fmt library
 
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_FMT libfmt QUIET)
+  set(FMT_VERSION ${PC_FMT_VERSION})
+endif()
+
+find_path(FMT_INCLUDE_DIR NAMES fmt/format.h
+                          PATHS ${PC_FMT_INCLUDEDIR})
+
+if (NOT ENABLE_INTERNAL_FMT AND NOT FMT_INCLUDE_DIR)
+  set(ENABLE_INTERNAL_FMT ON)
```

Comment:
Why auto fall-back to the internal build?
It should error if it is not found so user can then decide to install the (distribution) library or explicitly enable the internal build.

Reply:
Please read the discussion in this PR. I asked for guidance on how to handle this and @lrusak said I should do it this way.

Reply:
Using the distribution libraries is always preferred, except for libraries where Kodi requires a special version.
This isn't the case for this library so there is no reason for the auto fallback.

Reply:
For Ubuntu the situation with `libfmt-dev` is as follows:
* 16.04 doesn't provide it at all.
* 18.04 provides an older incompatible version.
* 19.04 provides an older incompatible version.
* 20.04 will provide a compatible version.

So except for Ubuntu 20.04 we must use the version provided by Kodi. But please discuss this with @lrusak since he suggested to do this auto-fallback. I don't have any opinion on depends builds since I usually work on Windows.

Reply:
I've already backported libfmt to all our supported ubuntu series.
There is no need for this fallback

Reply:
I reverted the whole commit. Now this PR only contains a fix for the expected version of `libfmt` / `Fmt`.